js/recipe/metadata.js

```diff
@@ -0,0 +1,31 @@
+import { htmlElement } from "../htmlRenderer.js"
+import { els } from "../htmlRenderer.js"
+import { render } from "../htmlRenderer.js"
+
+export function metadata(recipe) {
+  // Recipe Data
+  const tags = recipe.tags;
```

Comment:
Might be a nice candidate for object destructuring e.g.:
```js
export function metadata({ tags, datePublished, recipeYield, ...}) {
```
although it can make the parameter list a bit noisier.

Reply:
Yes! (just read up on Object destructuring), that is super neat!